ui/svelte.config.js

```diff
@@ -14,7 +14,10 @@ const config = {
 		// adapter-auto only supports some environments, see https://kit.svelte.dev/docs/adapter-auto for a list.
 		// If your environment is not supported or you settled on a specific environment, switch out the adapter.
 		// See https://kit.svelte.dev/docs/adapters for more information about adapters.
-		adapter: adapter()
+		adapter: adapter(),
+		alias: {
+			'@src/*': 'src/*'
```

Comment:
This is cool.

Reply:
It allows us to use paths from `src` in our TS script tags. No more relative imports :woot: